app/models/profile.rb

```diff
@@ -14,6 +14,11 @@ class Profile < ApplicationRecord
 
   scope(:active, -> { where(active: true) })
   scope(:verified, -> { where.not(verified_at: nil) })
+  scope(
+    :in_person_verification_pending, -> {
+      where.not(in_person_verification_pending_at: nil)
+    }
+  )
```

Comment:
Some think `scope`s should be avoided in favor of class methods. Happy to change these if we think so too.

Reply:
changed to class methods